poreCov.nf

```diff
@@ -113,6 +113,8 @@ if (params.fasta == true) { exit 5, "Please provide a fasta file via [--fasta]"
 if (params.fastq == true) { exit 5, "Please provide a fastq files (one per sample) via [--fastq]" }
 if (params.fastq_pass == true) { exit 5, "Please provide a fastq_pass dir via [--fastq_pass]" }
 if (params.fast5 == true) { exit 5, "Please provide a fast5 dir via [--fast5]" }
+if (!params.minLength == false && !params.minLength.toString().matches("[0-9]+")) { exit 5, "Please provide an integer number (e.g. 300) as minimal read length via [--minLength]" }
```

Comment:
* please change to `params.minLength`

Reply:
Changed
Exit-code also prevents user from using the [--min/maxLength]-flag without handing over a value